runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -7363,7 +7363,7 @@ reservationLockEnter(TR::Node *node, int32_t lwOffset, TR::Register *objectClass
       // BRC   MASK6, callHelper
       // #IF 64-Bit and JAVA_VERSION >= 19
       //    BRC   incrementOwnedMonitorCountLabel
-      // #ELSEIF
+      // #ELSE
```

Comment:
I see this on line 4140:
```
      //   #ENDIF
```
Can you add missing instances of that too, please?

Reply:
Looking at line 4140 [1], comment seems OK to me.

[1]. https://github.com/eclipse-openj9/openj9/blob/b28092a380bc55418fd8b1816c25dcf8ab13cb00/runtime/compiler/z/codegen/J9TreeEvaluator.cpp#L4134-L4146

Reply:
Yes, 4140 is fine. My suggestion was that there should be `#ENDIF` comments to match each `#IF` (e.g. for the ones on lines 7364, 7375, 7557, etc.).

Reply:
For example, lines 7364-7367 are:
```
      // #IF 64-Bit and JAVA_VERSION >= 19
      //    BRC   incrementOwnedMonitorCountLabel
      // #ELSEIF
      //    BRC   returnLabel
```
I don't know whether they should be
```
      // #IF 64-Bit and JAVA_VERSION >= 19
      //    BRC   incrementOwnedMonitorCountLabel
      // #ENDIF
      //    BRC   returnLabel
```
or
```
      // #IF 64-Bit and JAVA_VERSION >= 19
      //    BRC   incrementOwnedMonitorCountLabel
      // #ELSE
      //    BRC   returnLabel
      // #ENDIF
```
The indentation suggests the latter: we shouldn't be required to guess.

Reply:
This just fall of the radar, I apologize for that. 
Made changes to the all places where I added in the comment to clarify the flow in https://github.com/eclipse-openj9/openj9/compare/ce026d830e96edf18a89805291a78d97acf1f959..ec050eb523cd85dce5bc73226b6587266e9bf0f7

Reply:
We should have a space line 4145 after `->` to be consistent.

Lines 8120-8227 should be (the condition should use `== true` instead of `=` and the `#ENDIF` is missing):
```
             /*   #IF NextTest = GoToFalse
              *      branchCond = ifInstanceOf ? (!trueFallThrough ? COND_BE : COND_BNE ) : (init == true ? COND_BE : COND_BNE )
              *      branchLabel = ifInstanceOf ? (!trueFallThrough ? trueLabel : falseLabel ) : doneLabel
              *      CGRJ castClassReg, objClassReg, branchCond, branchLabel
              *   #ELSE
              *      CGRJ castClassReg, objClassReg, COND_BE, trueLabel
              *   #ENDIF
              */
```
It might read better of the ternary expressions were inverted, and the `== true` part is redundant.

Reply:
This does not appear to have been addressed.